flexibee-core/src/main/java/com/adleritech/flexibee/core/api/RetrofitClientFactory.java

```diff
@@ -22,20 +24,25 @@
 import java.util.Arrays;
 import java.util.concurrent.TimeUnit;
 
+@Component
 class RetrofitClientFactory {
```

Comment:
Namísto statických metod

Reply:
Jako jo, ale neni jasne co na nich v tomhle PR vadilo.

Reply:
To že FlexibeeClient měl konstruktory s credentials do Flexibee a v rámci konstruktoru volal tyhle statické metody a vytvářel Retrofit API. Takže v něm nešlo tohle API nahradit.